.coveragerc

```diff
@@ -4,7 +4,6 @@ source=skbeam
 omit =
     */python?.?/*
     */site-packages/nose/*
-    *test*
```

Comment:
Yeah, I had the same thought, and there was some discussion about this (which now I can't find...). The gist of is, including the tests in coverage is an extra way to help you notice if they aren't being _run_. Philosophically, I don't feel like tests themselves should count as "coverage". But given that we just ran a couple months without Travis even running the tests, CJ has a practical point here.